Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/MSBuildExecution/BuildLog.cs

```diff
@@ -48,10 +48,10 @@ public BuildLog(string filePath)
             root.VisitAllChildren<Build>(ProcessBuild);
             root.VisitAllChildren<Target>(ProcessTarget);
             root.VisitAllChildren<Task>(x => tasks.Add(x.Name));
-            root.VisitAllChildren<Message>(x => Messages.Add(x.Text.Replace("%24", "$")));
-            root.VisitAllChildren<Warning>(x => Warnings.Add(x.Text.Replace("%24", "$")));
-            root.VisitAllChildren<Error>(x => Errors.Add(x.Text.Replace("%24", "$")));
-            root.VisitAllChildren<Property>(x => properties[x.Name] = x.Value.Replace("%24", "$"));
+            root.VisitAllChildren<Message>(x => Messages.Add(x.Text));
```

Comment:
[learning] what is this about?

Reply:
The BinaryLogger messes up if there are non-english UTF8 characters. So, after migrating to the GCP machines the user running the test had a $ in the name which was sometimes logged correctly as a $ and at some points as %24. So, as a workaround the %24 was replaced to $.